core/controllers/question_editor.py

```diff
@@ -170,10 +170,6 @@ def get(self, question_id):
         question = question_services.get_question_by_id(
             question_id, strict=False)
 
-        if question is None:
```

Comment:
This was removed since this same check is now added to the decorator `can_view_question_editor`.